src/core_plugins/kibana/server/routes/api/scripts/register_languages.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash';
+import handleESError from '../../../lib/handle_es_error';
+
+export function registerLanguages(server) {
+  server.route({
+    path: '/api/kibana/scripts/languages',
+    method: 'GET',
+    handler: function (request, reply) {
+      const boundCallWithRequest = _.partial(server.plugins.elasticsearch.callWithRequest, request);
+
+      return boundCallWithRequest('cluster.getSettings', {
```

Comment:
Just curious, but why not just `server.plugins.elasticsearch.callWithRequest(request, 'cluster.getSettings`, { ...`

I've seen this pattern elsewhere in Kibana as well, but not quite sure why we create a new function.